packages/core/core/src/resolveOptions.js

```diff
@@ -49,7 +49,15 @@ export default async function resolveOptions(
     entries = [path.resolve(inputCwd, initialOptions.entries)];
   }
 
-  let entryRoot = getRootDir(entries);
+  let entryRoot = getRootDir(
+    entries.map(entry =>
+      // if the entry refers entryRootDir, getRootDir returns the parent directory of entryRootDir.
+      // In this case, each entry should refers some file, but without cli build target entry refers entryRootDir directory.
+      // So, we add /. to entry to make entry refers some (virtual) file in such case.
+      path.extname(entry) !== '' ? entry : entry + '/.',
```

Comment:
This would break if you have extensionless entries (which should work fine right now if there's a matching glob in parcelrc).
So maybe something like this instead?
```js
        (await inputFS.stat(entry)).isFile() ? entry : path.join(entry, 'index'),
```
(and use `path.join(entry, 'index')` instead of `./`. This pattern is used in various places, for example also when determining the projectRoot at few lines further down)

Reply:
Thank you! I fixed in [4b2d4a6](https://github.com/parcel-bundler/parcel/pull/7537/commits/4b2d4a6934967dfe511e64fed412e97c073600d8) 👍